Remotion/Web/Core/UI/Controls/ControlItemCollection.cs

```diff
@@ -279,13 +280,13 @@ public void Dispatch (IDictionary values, Control? parent, string collectionName
         page = parent.Page!.ToString()!;
       }
 
-      foreach (DictionaryEntry entry in values)
+      foreach (var entry in values)
       {
-        string id = (string) entry.Key;
+        string id = entry.Key;
 
         IControlItem? item = Find (id);
         if (item != null)
-          ResourceDispatcher.DispatchGeneric (item, (IDictionary) entry.Value!); // TODO: not null assertion
+          ResourceDispatcher.DispatchGeneric (item, entry.Value); // TODO: not null assertion
```

Comment:
```suggestion
          ResourceDispatcher.DispatchGeneric (item, entry.Value);
```

I think NRT and the WebString value type take care of the null-assertion.